pkgs/applications/video/dvb-apps/default.nix

```
@@ -0,0 +1,64 @@
+{ lib
+, stdenv
+, fetchurl
+, perl
+, fetchpatch
+,
+}:
+stdenv.mkDerivation {
+  name = "dvb-apps-3d43b280298c";
```

```suggestion
  pname = "dvb-apps";
  version = "3d43b280298c";
```

Is there a better version, however?

---

That is the same code (the date matches the hash), as you can see here. https://www.linuxtv.org/hg/dvb-apps/

I will change the version number in the nix to "1.1.1.20140321" to match Gentoo.